Assets/Tests/Utils/ECSTestBase.cs

```diff
@@ -13,6 +13,7 @@ public abstract class ECSTestBase {
   [SetUp]
   public void SetUpBase() {
     w = new World("Default");
+    World.DefaultGameObjectInjectionWorld = w;
     em = w.EntityManager;
```

Comment:
I think it would be better to name these something understandable like testWorld or just world.

Reply:
fixed in 6c9e816